NoSuchFileException when walking classes

Some plugins (like KSP) may create classes dirs but not generate anything into them. In these cases I think the class walking should omit those dirs if they don't exist. Happy to send a PR if this seems reasonable.

```
Caused by: java.lang.RuntimeException: java.nio.file.NoSuchFileException: /Users/zacsweers/dev/slack/android3/path/to/subproject/build/generated/ksp/release/classes
        at com.slapin.napt.ClassesProvider.walk(ClassesProvider.java:52)
        at com.slapin.napt.ClassesProvider.getClassNames(ClassesProvider.java:41)
        at com.slapin.napt.NaptPlugin$1.beforeAnnotationProcessing(NaptPlugin.java:21)
        at com.slapin.napt.AnnotationProcessingListener.started(AnnotationProcessingListener.java:21)
        at jdk.compiler/com.sun.tools.javac.api.ClientCodeWrapper$WrappedTaskListener.started(ClientCodeWrapper.java:843)
        at jdk.compiler/com.sun.tools.javac.api.MultiTaskListener.started(MultiTaskListener.java:119)
        at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.initProcessAnnotations(JavaCompiler.java:1122)
        at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:902)
        at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.lambda$doCall$0(JavacTaskImpl.java:104)
        at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.invocationHelper(JavacTaskImpl.java:152)
        ... 30 more
Caused by: java.nio.file.NoSuchFileException: /Users/zacsweers/dev/slack/android3/path/to/subproject/build/generated/ksp/release/classes
        at com.slapin.napt.ClassesProvider.walk(ClassesProvider.java:49)
        ... 39 more```

--- a/plugins/javac/src/main/java/com/slapin/napt/ClassesProvider.java
+++ b/plugins/javac/src/main/java/com/slapin/napt/ClassesProvider.java
@@ -34,6 +34,7 @@ class ClassesProvider {
                     return file;
                 })
                 .map(Paths::get)
+                .filter(Files::exists)
                 .flatMap(ClassesProvider::walk)
                 .map(Path::toString)
                 .filter(it -> it.endsWith(".class") && !it.contains("$"))

